examples/horovod/tf_hvd_simple.py

```diff
@@ -0,0 +1,125 @@
+# External dependencies
+import argparse
+import glob
+import os
+
+# we can control how much memory to give tensorflow with this environment variable
+# IMPORTANT: make sure you do this before you initialize TF's runtime, otherwise
+os.environ["TF_XLA_FLAGS"] = "--tf_xla_enable_xla_devices"
+# os.environ["CUDA_VISIBLE_DEVICES"] = os.environ.get("OMPI_COMM_WORLD_LOCAL_RANK")
+# TF will have claimed all free GPU memory
```

Comment:
Looks like a few lines got scrambled here, this comment line is part of the `IMPORTANT` note above.